update.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+git show "master:nsxiv.1" | groff -mandoc -Thtml > index.html
```

Comment:
We don't really need to store `index.html` , it can be piped into the awk command directly.